README.md

```diff
@@ -0,0 +1,3 @@
+# .github
+
+Public profile for HSE project organization on GitHub
```

Comment:
Public profile for the HSE project on GitHub

Maybe lost the organization since it seems encompassed by project.

Reply:
Agreed